twarep/electrifly
# Created SOH Graphs (SOH vs. SOC Rate of Change and Avg SOH per Month)

## Checklist

## What I Did
What files git you edit and why/for what purpose?
- app.py: to display the graphs
- Graphing.py: to create the graphs
- flight_querying.py: to query the data needed for the graphs

## How I Did it?
For the queries, I edited the get_flight_data_every_half_min_on_id query to account for SOH, and removed instances where SOH is 0 (which is the first row of each flight). I also added a new query called get_avg_soh_per_month_act_view which gets the avg SOH per month for ALL flights and also removed instances where SOH is 0.

## How to Test?
Manual Testing: go to statistical insights and see that the SOH vs. SOC rate of change graph is there, as well as avg SOH per month (note that avg SOH per month graph contains all data and isn't affected by the filters)
